myapp/app/views/tasks/_form.html.erb

```diff
@@ -13,27 +13,28 @@
 
   <div class="field mb-3">
     <%#= form.label :name %>
-    <%= form.text_field :name, class: "form-control", placeholder: "Task Name" %>
+    <%= form.text_field :name, required: true, class: "form-control", placeholder: "Task Name" %>
   </div>
 
   <div class="field mb-3">
     <%#= form.label :desc %>
-    <%= form.text_area :desc, class: "form-control", placeholder: "Description"%>
+    <%= form.text_area :desc, required: true, class: "form-control", placeholder: "Description"%>
   </div>
 
   <div class="field mb-3">
     <%#= form.label :status %>
-    <%= form.number_field :status, class: "form-control", placeholder: "Status"%>
+<!--    status_select, {selected: @task.status},-->
+    <%= form.select :status, Task.statuses, {selected: @task.status}, required: true, class: "form-control"%>
   </div>
 
   <div class="field mb-3">
     <%#= form.label :label %>
-    <%= form.text_field :label, class: "form-control", placeholder: "Label" %>
+    <%= form.text_field :label, required: true, class: "form-control", placeholder: "Label" %>
```

Comment:
[suggestion]
hmm. 
Since your model validator also didn't check, maybe we didn't need to this field to be required. (well that depend on how you want your apps to be like. I will leave that to your decision)

Reply:
actually i will add model validations in next step, so that's why added this.